torch_tvm/custom_tvm_ops/relay/quantize.cc

```diff
@@ -0,0 +1,148 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+ *  Copyright (c) 2018 by Contributors
+ * \file nn.cc
+ * \brief Property def of nn operators.
+ */
+
+#include <tvm/data_layout.h>
+#include <tvm/relay/op.h>
+#include <vector>
+
+#include <tvm/expr.h>
+#include <tvm/operation.h>
+#include <relay/op/op_common.h>
+#include <relay/op/type_relations.h>
+#include <relay/pass/alter_op_layout.h>
+#include "quantize_attrs.h"
+
+#include <cstdint>
+namespace tvm {
+namespace relay {
+
+Expr MakeDataInt8Quantization(Expr data, Expr zero_point, Expr scale, bool is_signed, int precision) {
```

Comment:
What is the use of precision here? It seems Int8 is already telling? Or is it for output precision?

Reply:
hmm, just in case, for 16bits, for example.